subtraction.c

```diff
@@ -0,0 +1,5 @@
+#include "basic_operation.h"
+
+int subtraction(int a, int b) {
```

Comment:
```suggestion
int sub(int a, int b) {
```

This function's name does not match with the header file's.